pkg/operator/options/suite_test.go

```diff
@@ -107,6 +108,7 @@ var _ = Describe("Options", func() {
 				SSHPublicKey:                   lo.ToPtr("env-ssh-public-key"),
 				NetworkPlugin:                  lo.ToPtr("env-network-plugin"),
 				NetworkPolicy:                  lo.ToPtr("env-network-policy"),
+				SubnetID:                       "/subscriptions/12345678-1234-1234-1234-123456789012/resourceGroups/sillygeese/providers/Microsoft.Network/virtualNetworks/karpentervnet/subnets/karpentersub",
```

Comment:
Just getting lucky here, current test.Options implementation does not allow override, won't work with any other value?